docs/test-analysis-cli-orphan-detector.md

```diff
@@ -0,0 +1,78 @@
+# AppHostExitsWhenCliProcessPidDies Test Analysis
```

Comment:
Why is this getting committed?

Reply:
Removed it myself - faster in this instance.